pkg/connector/database_users.go

```diff
@@ -182,8 +182,36 @@ func (o *databaseUserBuilder) CreateAccount(ctx context.Context, accountInfo *v2
 		return nil, nil, nil, err
 	}
 
+	userTraits := []rs.UserTraitOption{
+		rs.WithUserProfile(
+			map[string]interface{}{
+				"username":      username,
+				"login":         username,
+				"database_name": databaseName,
+			},
+		),
+		rs.WithUserLogin(username),
+		rs.WithStatus(v2.UserTrait_Status_STATUS_UNSPECIFIED),
```

Comment:
Do we know if the created account is enabled? If so, we can set this to UserTrait_Status_STATUS_ENABLED.

Reply:
Make sense, I will change